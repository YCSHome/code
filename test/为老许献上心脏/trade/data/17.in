400 500 500 
278 104400040119931
199 358325234075557
218 278368949330133
16 513677460298597
292 697656183866541
223 748248641973660
383 723184712587382
161 645069505781411
364 794577041420158
247 828543227210269
206 405177141544622
369 140273392115694
275 596138567218215
6 135926220778353
153 145412068708119
97 785866459826868
223 144335131222864
217 687104524080906
210 925552902496700
354 216180732167165
201 445195349408478
238 532003703008060
2 355273876892517
209 858814454852809
154 347769417995921
377 11921781662292
56 354231906134623
13 932286412106197
260 822748681712200
246 892359807426946
391 478693168042368
302 708996827564619
376 855409733100334
396 625909427849272
233 758626452949574
238 358694933332105
198 503353619328858
106 748541783776874
217 856954307988930
128 752309608565664
126 979212222113103
297 212044595160076
102 610101258551039
316 67442171387229
125 805259129485903
99 147748176100985
236 716948585395609
44 620042360544128
39 651768219197122
253 775091264141571
168 845223130095392
111 884076351256040
389 600221085062333
215 275988639789469
203 13219024407998
54 37843375946214
31 868127284602872
67 46516154739025
295 246354107956999
12 603515652952694
148 685830341048032
290 190080254093012
32 657807610723071
241 372990075952088
271 333313265733982
243 140097062819544
307 189294096595501
346 980909500965685
176 79322628597301
148 259143185827329
279 28143961416874
71 89797289491820
191 125898117122447
182 437965381170240
64 753108144039425
286 763795028952354
292 456233769162721
189 899777674594839
118 545275508772081
141 423473431671120
184 132152404101033
280 589402622305537
188 412194299284520
395 224199529433182
202 218832032489515
192 133083532644959
126 659964770828326
206 745142690987302
203 389507719924282
130 265573328456274
183 542551234468673
178 533157214085943
50 366214972603304
328 795631625794512
123 347273548345137
191 556187710777771
331 816638478843690
286 337667528973681
30 572753371739071
10 716810107849702
75 358335438465956
138 876321090734204
95 943575649471677
149 449304599841240
136 758537422290086
28 156341122870168
255 64401069069898
52 643669084774567
361 451362081540660
127 89229245133512
135 719869770086159
23 588088770242701
96 326846635792188
286 352636797492419
172 761508225925497
329 426078533400868
388 780099573598317
82 707640413160893
164 88011201931070
392 911943613006046
253 804627427188560
248 764695157311781
91 634739461534641
29 536388076957046
99 934997501746056
87 445061569294145
108 561495191618929
299 782282072629614
30 447078440220718
170 726697401421624
345 9564461264146
77 626297839408748
118 454329471417825
178 410785150685072
21 964926593729031
196 162290794587261
227 314558479397441
382 816758514594138
125 254920870651831
72 548548237745268
329 703077509344766
266 285632170095937
203 600952245549627
283 606437845725974
399 663428201333071
214 676136389356088
98 507657013137004
117 986682996090669
35 679049920521617
264 269739981403806
352 598906281669212
284 246458563148162
47 732830340319579
216 777274839415426
68 33034621172948
98 792248871563585
10 93490616627518
379 967376551863822
58 951157888120450
327 92166640959609
380 467633564467792
29 253585197640885
363 669446216008258
284 63600549106575
217 59071651544792
7 369139236510182
193 227942384308004
184 854859867803722
324 689691187018490
166 215281491271122
109 680729489161298
262 923584110798493
113 526945864372379
176 784789797618876
358 528119318588026
384 713494853508726
210 797921562117608
88 12084596842924
171 436614242747669
256 976826907677780
335 758846021594122
53 894695576919669
254 804389387311700
76 470303053300905
282 211185473932371
121 347605012472927
203 15359601785011
154 136201705285975
106 469226265786418
191 956200838509026
381 887140740556850
399 915193342900692
153 1677700300813
188 497714238934948
208 262472982163808
19 329103060969229
129 50179732034639
45 601717277208065
255 173738400759656
191 100658013435251
193 325094102215733
79 921540816947899
246 992821648816326
15 437672983448283
354 239127399278852
342 887783838960021
356 911183988662073
306 771438456730319
9 192381133927227
261 203591540110019
182 585995754667960
183 857951416995735
118 718197910380888
61 340389459760750
275 832150956190483
231 462954515566660
247 803708489878661
65 773984986882316
331 396306087399763
142 937404522903023
397 516043635123722
127 611806964475670
258 349902534435837
204 642587114891973
188 875306427339882
347 469441963564132
118 397916071547362
210 684599047386736
74 32413146001041
313 594420347293954
130 283343337770677
4 750103343223372
4 878734553446815
288 163621887523392
63 528120510624095
3 443548808350925
306 904502199169539
175 418849979356077
167 770629503025443
288 287263919035226
358 42128049356281
351 143860334380161
325 94084394810550
287 21740692047989
68 552807961039689
167 808742740463964
208 641230567118725
155 38137957143630
177 555783401199487
186 265361250890822
34 118248194550673
142 493634595587105
227 974934088153104
258 777141159399521
264 6410463154205
59 636209967158155
370 524989114431206
364 30551361745373
49 98547870926595
197 541268755113779
48 532452797253586
256 877516499130031
26 351292606735082
335 430591186553204
303 486887850762712
353 903733562367841
354 136377164506736
18 355716739150030
98 144098000683685
246 636397534132324
380 664142832589790
94 569391975137297
44 151373406438792
302 618254186808551
101 248791776057314
387 523973842570388
99 636573827783792
226 716757384980776
324 224180565197359
192 632767903419279
155 931735575443365
110 915169313930222
206 561559255667412
214 406027565525714
115 806759703870245
70 424453485244102
201 839282387496691
195 837567185165194
94 698346291023368
193 242901601872683
126 160156512599239
148 449880095596413
147 20975055083192
204 291817209267954
43 579902850225972
154 845460933967661
205 317091623470440
237 562295293309935
9 136820496080316
314 916569001131430
368 867653231533714
213 435245966834228
371 293729183778866
149 683785493482279
230 243520856306389
384 292593349498504
115 434092609738358
192 303054719407677
37 455338559281118
101 527405518101123
325 359223121678751
23 735539991145957
56 654666285285285
216 262292893993351
113 162783792602971
156 206874971437619
323 68408874955372
107 307624709097402
95 284580554540840
345 653287984856466
273 202692842783575
265 290794029349278
67 807692706324671
178 512979053971189
63 633838039931526
115 901959666516982
320 860159170590813
33 292639262633326
138 339028993293498
312 791446758969731
185 211366044142774
69 435231857654403
389 930688573962796
247 205747509400209
323 736644411115580
311 316637369033733
301 504812032639660
212 541601247906958
368 528520651683861
184 266068115546120
188 496445969771789
48 170724041423080
291 505227338724085
180 480651110249402
217 960762495037431
163 442198005308930
373 276536663092328
246 596943228949412
83 491988144789155
14 31548678842165
269 898624889245723
206 137078365920757
92 544107861911636
19 699815617232280
183 573422553418244
29 44764037529978
67 264847768414713
85 317411996736684
252 251418890219275
155 525274548559593
149 262301316674107
94 227477056030354
338 294655880368087
280 162870905360164
273 507590000333722
18 383850909297608
359 77231630080827
251 712427652905921
38 547328103113384
127 644580730102693
340 563952993803184
96 316424750039387
9 616032413144321
41 676494396089864
241 133123105481748
231 108019983069138
215 283514347371897
349 860174112438649
126 968751848500924
305 173229754158239
321 235649249164278
104 582987204060077
366 518790864942158
342 614325529889021
199 443922394819273
351 144728283398195
8 586031256066577
73 568827844575817
308 118181401346938
154 585730218291823
148 303956135990694
395 27834631782529
4 130801737338371
243 160773889930023
88 562520168292030
54 886077507441617
118 732099439015034
125 433624999793132
278 323300548193538
70 594165488219472
185 576478082331986
394 468586651873405
3 85563128832238
395 236679612632965
292 25377925234311
9 74081561842852
266 807639561325210
54 146298116111874
382 693983434712868
21 544126342330252
246 709661021873635
51 801675108068040
368 428137092109213
18 592004490442329
365 801126293442291
367 995789037962037
262 31261733924267
178 56888925890962
336 990773046218944
127 896691492865936
372 815258460364414
220 57471838498965
268 538865446309140
380 87560106125894
246 952356711768981
209 406311855746598
78 514720304080697
262 464571127938061
89 273408677829888
82 629489372387875
172 809642345218758
174 426849293555092
117 185559550663777
281 649995182785996
369 28811265228187
133 461672159492842
85 824194156430203
36 31112994818528
54 963019413353613
18 222894282384834
95 918492113833171
237 748934509216654
157 447247945734122
278 717076045663323
107 353685830360141
126 930285544147769
296 952564534140302
104 628398120811509
311 504134818422963
223 235109131432508
55 530755085868475
246 970927764059685
332 977562126089982
11 380997829443800
342 679878302991525
169 668926388295510
212 32014756669321
391 747945574081151
334 892146826691337
247 435639437660678
335 773810541376533
285 510578713283625
328 670187135927807
230 263796372002740
2 971022236154368
364 683855311976739
220 69337241327349
132 95295331140456
175 32454913021378
180 535915346238589
270 414735945697185
186 215762557857745
94 415039340148685
185 140190909993300
138 819192939606038
192 111949245276641
230 783631380813604
79 564077332423240
368 922237898431583
295 321374070207168
147 798326205594542
179 964614808331825
43 387638777492804
350 584275953068015
15 893917416118430
155 535518224992781
102 967003768460701
161 384073281908884
152 893261911375502
222 877317733300761
141 675190723736229
171 358269972120796
120 410980513098832
286 822997630605843
113 36273458372600
127 166091132883448
275 747582356469148
257 255083879525143
226 931547948405686
176 725813761843948
120 722145407467407
293 730464784034677
163 996151107143735
31 568225998806444
7 897723141960936
210 292430113100218
73 489557901393080
288 597210044944461
156 437697725186800
324 371577597992941
362 103782058497709
248 743268773884955
155 112373338677905
355 798800459822854
275 937043087708677
325 805098393222530
122 526619853803389
317 243597311588969
155 739634230158117
128 239953679658389
198 542195383904144
314 4230001581759
286 51813896800667
398 968907240974579
279 522650410242556
121 789813900420097
55 496053991027
149 226691647646327
61 787511646227430
108 987306012618127
234 999608392621849
109 666027790312220
350 578090112301834
43 584016168316327
218 466572117624332
313 119475494101569
215 293729797767752
208 799763744446932
320 496668436075768
266 759077110038911
384 222679965969667
27 658288756218946
255 245505321871154
34 25526198621595
176 797460651539089
85 560079626425457
343 599695875280924
391 156080191310824
334 128742095303023
125 534419322554147
271 96151913922836
316 480371635063913
18 90335789310203
49 380011967515082
140 955204691759000
75 590537817815883
203 983722110664334
349 523270545136317
282 879794476838315
206 97742566903436
399 814628247369859
294 932869924626774
42 184566180043478
316 419102091099543
148 604336378618960
229 524159878646528
123 898986715187202
390 45564484421246
300 29851517049182
162 374335556380861
244 951531878283022
314 895921190808120
219 447950085756202
282 992012116134097
394 450748057336603
295 644538931972282
284 481827952196770
159 225452377205140
272 793656380105702
84 4240109136383
126 765941581342516
159 705243236023818
172 309353738078243
342 376663442079033
112 836226345877114
280 975246897611411
2 276615526415355
270 717188448047612
41 209467708387781
219 698172471561987
392 564953149614903
376 270101252438456
210 133003097728784
78 220636313579163
374 493693040717956
4 250444883093681
332 996399030867279
180 400013083862990
164 134442153750664
234 743273305904573
389 999615450235317
223 61355826718758
317 318960286850407
270 446450183902194
315 353249482391397
244 515904651041345
284 82087391536792
127 228383586801593
24 180977134713367
172 2367877367474
151 510536432129246
291 745078090899133
86 461460980845134
53 821146328283681
334 143912642088595
334 160080407069318
143 673831449640690
29 818433127021817
115 420472500124442
71 935356298596502
260 599470990624400
43 56930541894351
72 738044063350806
102 890071496603616
193 886638822104218
334 217071370816683
265 452556568874127
223 182697614429263
251 48310477138416
382 466879936705029
267 948446539660256
200 407136907708930
198 656056394428597
45 859263105856762
377 942359343673781
121 959048462761642
95 623467509859741
298 310189836527455
387 18523395772110
367 13242686524244
180 118855793271263
68 739450479792243
108 747625557066269
337 779490682186680
302 184102584403232
89 950450061323232
20 410048494170332
49 18144411535758
79 309183529569752
224 463820197968589
109 40469454212478
296 639974570830231
88 337236087671620
91 784409106703814
83 225137516317484
342 844147559964298
106 371528399147720
194 669132836043005
290 38135030950763
183 254592087881388
205 189478037048883
315 363947915156529
67 968780467328181
12 200761383705447
224 303275594035462
40 532450358985786
154 132679892045586
136 478086129922489
177 255751284253716
389 193430915990095
235 749984829018179
286 224318641977366
289 789614450461694
161 72591251708593
231 57810101290857
129 51582797833126
339 662847648795554
73 577375917747073
230 236125696457248
237 902376201968264
104 516084847271796
22 196483493805800
203 365619381358097
34 409937570170604
356 913620448592591
225 137488017504304
171 770281031543563
117 117081473036691
144 413553316275262
308 74354185563346
51 891471482640972
340 429701440702733
59 924836103912292
48 76940466766646
16 648287934375918
354 557547455635188
172 12167177993142
234 142573850936469
290 562005332449673
375 872373287807451
202 854052248934334
358 715427096832448
265 558127236884760
140 200016294592795
85 339116744416821
351 263407818926503
123 10798988635730
218 76199454961805
115 982509173137929
241 129523669754131
310 750240025899714
390 713626479752924
270 364770245584903
299 286876767991951
396 564256281477031
333 677917362202282
360 805615700081198
181 195021990443528
320 806027714606109
237 727460634404699
105 546675112863860
1 550081435210752
203 198219854792765
272 99562785341280
302 872596681114650
300 406372339478472
392 588811085739709
209 885714989367794
244 252746470546202
65 905024933409268
104 176715243436413
387 223816497078871
34 592476654120323
388 779413317924702
143 879799106672705
301 71282197891740
39 992377741604276
196 494404338717267
308 873258686361041
197 523641265861185
49 975211494829682
247 978594201503274
364 580292357396970
275 577640532993206
101 939128587905046
369 130093544397716
322 238483972982700
375 833352619760269
47 598949664215994
155 317421517052859
235 961599070055987
272 552697951436178
42 384033686947228
213 924294410980132
187 433058625118263
282 607301809355876
80 256490646236631
398 952668375254520
355 954036329237028
212 194510168443220
230 118060739219674
107 312529524216657
253 461429054489457
374 824822772647069
371 652104377953964
280 621982338952407
377 155701821089941
395 952379918584538
213 344128476531608
18 125496593178406
56 80603004090267
46 67914818226308
53 932505455293858
62 763808226415164
373 668942054723143
267 343518798362157
69 130741294717122
378 982534172203297
40 264534876915265
220 478407785830531
238 781918372827637
251 393136218956855
375 376117109859277
260 733719072733778
145 489473030810458
235 149499872710614
246 155577709934750
16 641174173415320
255 399533090164910
4 686442622593699
125 867504449897392
274 710373255464696
173 979714983542534
329 735231497619607
148 390609103774499
284 65685304630752
172 144574390726408
386 276243405640673
202 120507832212628
23 52495220677088
