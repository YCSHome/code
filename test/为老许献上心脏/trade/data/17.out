8830423453526276
